calc.py

```diff
@@ -1,14 +1,20 @@
-def calc():
-	# Write code main function
+def calc(): 
+    while 1:
+        num1, num2, func = input("Input Command (ex:1,2,'+') : ") 
+        if (func == '+') : 
+            ret = add(num1, num2) 
+        elif (func == '-') : 
+            ret = sub(num1, num2) 
+        else : 
+            print('Wrong Command!') 
+            ret = None 
 
-def add(num1, num2):
-	# Write code add function
-
-def sub(num1, num2):
-	# Write code sub function
+ret = calc()

```

Comment:
Actually it returned ret. You can see that ret is the thing you print on the line 14.